editoast/editoast_models/src/tables.rs

```diff
@@ -409,6 +409,26 @@ diesel::table! {
     }
 }
 
+diesel::table! {
+    use diesel::sql_types::*;
+    use postgis_diesel::sql_types::*;
+
+    macro_node (id) {
+        id -> Int8,
+        scenario_id -> Int8,
+        position_x -> Int8,
+        position_y -> Int8,
+        #[max_length = 255]
+        full_name -> Nullable<Varchar>,
+        connection_time -> Int8,
+        labels -> Array<Nullable<Text>>,
+        #[max_length = 25]
+        trigram -> Nullable<Varchar>,
```

Comment:
This contains more than just the trigram: this also contains the secondary code. To avoid mixing up unrelated fields, it would be best to rename this to something else. Maybe `trigram_with_code`?